configure.py

```diff
@@ -471,7 +471,7 @@ def nw4rLib(lib_name, objects, extra_cflags=[]):
             Object(NonMatching, "d/col/bg/d_bg_w_kcol.cpp"),
             Object(Matching, "d/col/bg/d_bg_w_sv.cpp"),
             Object(NonMatching, "d/col/bg/d_bg_w_time.cpp"),
-            Object(Matching, "d/col/cc/d_cc_shape_colliders.cpp"),
+            Object(NonMatching, "d/col/cc/d_cc_d.cpp"),
```

Comment:
Is there a particular reason this file goes from Matching to NonNatching?

Reply:
Oh oops, this should be good again. I forget I changed it to non matching when transferring it